ballerina/Ballerina.toml

```diff
@@ -4,14 +4,14 @@ name = "io"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
-path = "../io-native/build/libs/io-native-@project.version@.jar"
+path = "../native/build/libs/io-native-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "io"
 version = "@project.version@"
 
 
 [[platform.java11.dependency]]
-path = "../io-test-utils/build/libs/io-test-utils-@project.version@.jar"
+path = "../test-utils/build/libs/io-test-utils-@project.version@.jar"
 groupId = "org.ballerinalang.stdlib.io"
```

Comment:
Don't we need to update this?

Reply:
Since the path is given, we can actually remove the rest of the params except scope.

Reply:
Updated with https://github.com/ballerina-platform/module-ballerina-io/pull/252